Library/Formula/boxes.rb

```diff
@@ -17,7 +17,7 @@ def install
 
     # distro uses /usr/share/boxes change to prefix
     system "make",
-      "GLOBALCONF=#{share}/boxes-config",
+      "GLOBALCONF=#{pkgshare}/boxes-config",
```

Comment:
I think that we should handle the edge case, what do you think? Also there is another one in ```arm```
```diff
diff --git a/Library/Formula/arm.rb b/Library/Formula/arm.rb
index e387899..a9fc6f6 100644
--- a/Library/Formula/arm.rb
+++ b/Library/Formula/arm.rb
@@ -7,8 +7,8 @@ class Arm < Formula
   bottle :unneeded

   def install
-    (share+"arm").install Dir["*"]
-    bin.write_exec_script share/"arm/arm"
+    pkgshare.install Dir["*"]
+    bin.write_exec_script pkgshare/"arm"
   end

   def caveats; <<-EOS.undent
```

```
==> audit problems
arm:
 * A `test do` test block should be added
 * Use pkgshare instead of (share/"arm")
 * Use #{pkgshare} instead of #{share}/arm

Error: 3 problems in 1 formula
```